source/D0323r5.bs

```diff
@@ -363,18 +350,17 @@ deleted unless `is_default_constructible_v<T>` or `T` is `void`.
 constexpr expected(const expected& rhs);
 ```
 
-*Effects*: If `rhs` contains a value, initializes the contained value as if
+*Effects*: If `bool(rhs)`, initializes `val` as if
 direct-non-list-initializing an object of type `T` with the expression `*rhs`
 (if `T` is not `void`).
 
-If `rhs` does not contain a value initializes the unexpected value as if
+If `! bool(rhs)`, initializes `val` as if
```

Comment:
Extra space after `!`.

Reply:
Should I replace all the `! bool` by `!bool`

Reply:
Yes. I think that's what the standard does.